docs/source/index.rst

```diff
@@ -17,13 +17,20 @@ low operational overhead.
 Organizations can use Custodian to manage their cloud environments by
 ensuring compliance to security policies, tag policies, garbage
 collection of unused resources, and cost management from a single
-tool. Custodian policies are written in simple YAML configuration
-files that specify given resource types and are constructed from a
-vocabulary of filters and actions. Custodian adheres to a compliance
+tool. 
+
+YAML Policies are the core concept of Cloud Custodian.  They take actions 
```

Comment:
the wording here feels akward.. yaml policy/config files aren't really the core concept, there just the expression of configuration. The core concept is a stateless rule engine, that filters and takes actions on resources, and can be bound to serverless event streams across multiple cloud providers.

Reply:
Agreed - I'll take a pass at updating that.

Reply:
The core concept is a stateless rule engine, that filters and takes actions on resources. It can be bound to serverless event streams across multiple cloud providers that maps to security, operations, and governance use cases. Custodian adheres to a compliance as code principle, so you can validate, dry-run, and review changes to your policies. 

Cloud Custodian policies are expressed in YAML and include the following:

* The type of resource to run the policy against
* Filters to narrow down the set of resources
* Actions to take on the filtered set of resources